mongodb-odbc-driver/bin/build-iodbc.sh

```diff
@@ -6,7 +6,12 @@
 	mkdir -p "$BUILD_DIR"
 	cd "$BUILD_DIR"
 	echo "downloading iODBC"
-	curl -O "http://noexpire.s3.amazonaws.com/sqlproxy/binary/linux/$IODBC_VERSION.tar.gz"
+	curl -O "http://noexpire.s3.amazonaws.com/sqlproxy/binary/linux/$IODBC_VERSION.tar.gz" \
+       --silent \
+       --fail \
+       --max-time 60 \
+       --retry 5 \
+       --retry-delay 0
```

Comment:
setting `--retry-delay 0` turns off exponential backoff. Are we sure we want to do that?

I'd personally set max-time up to about 300 and let the exponential back off do its thing, but maybe you have some reasoning I haven't though of? :)

Reply:
I think there were discussion before? https://github.com/10gen/sqlproxy/pull/105#discussion_r280861562 it does use exponential back off when --retry-delay is set to 0